_posts/2022-09-19-ocp-prod-network-maintenance.md

```diff
@@ -0,0 +1,14 @@
+---
+title: ocp-prod Cluster Network Maintenance
+severity: 0
```

Comment:
Do you plan to set this to 1 or more on the day of the outage?

Reply:
correct, we want it as a heads up until 8 am Monday then I will change it to 1 and then remove the active status once the maintenance is done.